src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md

```diff
@@ -1,55 +1,118 @@
 ---
 title: Scheduled Jobs with Custom Clock Processes
 nav: Custom Clock Processes
-modified_at: 2021-10-19 10:00:00
+modified_at: 2023-02-01 17:00:00
 tags: task-scheduling
 index: 2
 ---
 
-With custom clock processes, you have the ability to specify custom schedule. You will also have other benefits like
-environment parity between development and production, or time precision in your task scheduling.
+Scalingo provides a feature called *Custom Clock Processes* to help you run
+tasks on a regular basis. Unlike jobs setup through the [Scalingo Scheduler]({% post_url platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md %}),
+custom clock processes do not suffer from many limitations.
```

Comment:
```suggestion
custom clock processes do not suffer from the same limitations.
```

Reply:
Fixed in https://github.com/Scalingo/documentation/pull/1935/commits/a7a03d02828f4da629fa74557a6f3ed9c866f9c6